test/e2e/cypress/fixtures/hana-cluster-details/available_hana_cluster.js

```diff
@@ -168,7 +172,7 @@ export const availableHanaCluster = {
             {
               id: 'rsc_SAPHana_HDP_HDB10',
               type: 'ocf::suse:SAPHana',
-              role: 'Slave',
+              role: 'master',
```

Comment:
I think this is a mistaken change. This value doesn't have anything to see with the current implementation.
I don't know if in a real cluster though :see_no_evil:

Reply:
yeah, too much ctrl+d in vscode, reverted 👀